odoosentinel/__init__.py

```diff
@@ -126,6 +132,23 @@ def scanner_check(self):
         if isinstance(self.scenario_id, list):
             self.scenario_id, self.scenario_name = self.scenario_id
 
+    def _resize(self, initial_screen_size):
+        """
+        Resizes the window
+        """
+        # Asks for the hardware screen size
```

Comment:
Instead of an inline comment, this should be explained in the "elaborate description lines" (as described in PEP-257 multi-line section) of the docstring above